dataprocspawner/spawner.py

```diff
@@ -330,6 +347,32 @@ def _options_form_default(self):
       html_customize_cluster
     ])
 
+  def _list_gcs_files(self, gcs_files_path, extension="", return_path=True):
+    """ Lists the file names of a GCS bucket or subfolder.
+    Args:
+      - str gcs_files_path: Represents a bucket or subfolder in GCS. Can include
+        gs:// and a trailing /.
+      - str extension: if provided, includes only the files that ends with this
+        string.
+    """
+    try:
+      gcs_files_path = self._clean_gcs_path(gcs_files_path, return_gs=False)
+      gcs_files_split = gcs_files_path.split('/')
+      gcs_bucket = gcs_files_split[0]
+      gcs_subfolders = '/'.join(gcs_files_split[1:])
+      blobs = self.gcs_client.list_blobs(gcs_bucket, prefix=gcs_subfolders)
+      # prefix_path = gcs_files_path if return_path else ""
```

Comment:
Delete?

Reply:
Fixed in db78ebd3a93d1f4f16a14204a172dc22e1b8df99